config.py

```diff
@@ -0,0 +1,95 @@
+import json
+
+
+def get_color():
+    try:
+        return get_json_file_setting("Color")
+    except IOError:
+        print("Could not retrieve information from Json file")
+        return "Red"
+
+
+def set_color(color, hex_value=None):
+    try:
+        update_json_file_setting("Color", color)
+        if(hex_value is not None):
+            update_json_file_setting("Hex Color", hex_value)
+    except IOError:
+        print("Could not update Json file")
+
+
+def get_hex_color():
+    try:
+        return get_json_file_setting("Hex Color")
```

Comment:
For json keys, lets follow camel case, also putting space in key names is not a good practice. So please change it to HexColor.